### PR TITLE
[Mostly editorial] Merge BidirectionalStreamsTransport into WebTransport

### PR DESCRIPTION
**Pull request by yutakahirano**
This is similar to #258.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webtransport/pull/261.html" title="Last updated on May 13, 2021, 5:36 AM UTC (403ea58)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webtransport/261/91ae9d6...403ea58.html" title="Last updated on May 13, 2021, 5:36 AM UTC (403ea58)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -791,6 +714,18 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
...
    The getter steps for the `datagrams` attribute SHALL be:
      1. Return [=this=]'s [=[[Datagrams]]=].
+: <dfn for="WebTransport" attribute>incomingBidirectionalStreams</dfn>
+:: Returns a {{ReadableStream}} of {{BidirectionalStream}}s that have been
+   received from the remote host.
+
+   The getter steps for the `incomingBidirectionalStreams` attribute SHALL be:
+
+     1. Return [=[[ReceivedBidirectionalStreams]]=].
+     1. For each bidirectional stream received, create a corresponding
```
Comment (ricea) [inline index.bs:724]: It's weird that this step is here. The getter should have no side-effects, so bidirectional streams should be added to [[ReceivedBidirectionalStreams]] regardless of whether the getter was accessed.

Anyway, this is a pre-existing problem so no need to fix it in the this PR.

File: index.bs [hunk 2]
```diff
@@ -819,6 +754,47 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
          1. Once stats have been gathered, resolve |p| with the
             {{WebTransportStats}} object, representing the gathered stats.
 
+: <dfn for="WebTransport" method>createBidirectionalStream()</dfn>
+:: Creates a {{BidirectionalStream}} object for an outgoing bidirectional
+   stream.  Note that in some transports, the mere creation of a stream is not
```
Comment (ricea) [inline index.bs:759]: Maybe remove "in some transports" ?

**Review by ricea — APPROVED** [review]
lgtm

File: index.bs (hunk 2 above)
Comment (yutakahirano) [reply index.bs]: Done.

File: index.bs (hunk 1 above)
Comment (yutakahirano) [reply index.bs:724]: Yeah, I agree. Fixing it would be easier with having this in WebTransport interface.

**Pull request merged**